Dockerfile

```diff
@@ -44,27 +44,18 @@ COPY --from=vendored /out /
 # After building with jekyll, fix up some links
 FROM gem AS generate
 ARG JEKYLL_ENV
-ARG DOMAIN
+ARG DOCS_URL
 ENV TARGET=/out
 RUN --mount=type=bind,target=.,rw \
   --mount=type=cache,target=/src/.jekyll-cache <<EOT
 set -eu
-if [ "${JEKYLL_ENV}" = "production" ]; then
-  (
-    set -x
-    bundle exec jekyll build --profile -d ${TARGET} --config _config.yml,_config_production.yml
-    sed -i 's#<loc>/#<loc>https://${DOMAIN}/#' "${TARGET}/sitemap.xml"
-  )
-else
-  (
-    set -x
-    bundle exec jekyll build --trace --profile -d ${TARGET}
-    mkdir -p ${TARGET}/js
-    echo '[]' > ${TARGET}/js/metadata.json
-  )
-fi
+CONFIG_FILES=_config.yml$([ "$JEKYLL_ENV" = "production" ] && echo ",_config_production.yml" || true)
+(
+  set -x
+  bundle exec jekyll build --profile -d ${TARGET} --config ${CONFIG_FILES}
+)
 find ${TARGET} -type f -name '*.html' | while read i; do
-  sed -i 's#\(<a[^>]* href="\)https://${DOMAIN}/#\1/#g' "$i"
+  sed -i 's#\(<a[^>]* href="\)${DOCS_URL}/#\1/#g' "$i"
```

Comment:
@thaJeztah About what you said in https://github.com/docker/docker.github.io/pull/14784#discussion_r878382327, do we want to tackle this here or in a follow-up?

Reply:
Probably best to not change it (for now) until we have a full understanding of consequences (if we _would_ change)?

Reply:
`DOCS_URL` is needed for `update_sitemap.rb` plugin but here it doesn't change the current behavior on prod and will still be `https://docs.docker.com`